editor-components/feature/feature7/feature7.tsx

```diff
@@ -153,17 +153,17 @@ class Feature7 extends BaseFeature {
   }
 
   render() {
-    const hasTitle = this.castToString(this.getPropValue("title"));
-    const hasMainTitle = this.castToString(this.getPropValue("mainTitle"));
-    const hasdescription = this.castToString(this.getPropValue("description"));
-    const hasFeatures = this.getPropValue("features");
-    const hasLinks = this.getPropValue("links");
+    const title = this.castToString(this.getPropValue("title"));
+    const mainTitle = this.castToString(this.getPropValue("mainTitle"));
+    const description = this.castToString(this.getPropValue("description"));
+    const features = this.getPropValue("features");
+    const links = this.getPropValue("links");
     const hasTextContent =
```

Comment:
check line 179. you are using "hasTextContent.length". It has a bit of a strange spelling